### PR TITLE
Add comments for fields in the response for nodejs/query endpoint

### PR DESCRIPTION
**Pull request by chejennifer**
This came up because Kara is adding documentation about the nodejs/query endpoint & wanted clarification about some of the fields in the response. Thought this information could be useful to have here

### CODE REVIEW

File: static/nodejs_server/types.ts [hunk 1]
```diff
@@ -60,18 +60,29 @@ export interface TileResult {
 
 // Properties to use for drawing a single chart
 export interface ChartProps {
+  // Dcid of the place shown in the chart. If it's a chart that shows children
```
Comment (kmoscoe) [inline static/nodejs_server/types.ts:63]: use DCID (all uppercase throughout)

Comment (kmoscoe) [inline static/nodejs_server/types.ts:63]:
```suggestion
  // DCID of the place shown in the chart. If it's a chart that shows child
```

File: static/nodejs_server/types.ts [hunk 2]
```diff
@@ -60,18 +60,29 @@ export interface TileResult {
...
+  // places such as map or ranking, this is the dcid of the parent place.
```
Comment (kmoscoe) [inline static/nodejs_server/types.ts:64]:
```suggestion
  // places such as map or ranking, this is the DCID of the parent place.
```

File: static/nodejs_server/types.ts [hunk 3]
```diff
@@ -60,18 +60,29 @@ export interface TileResult {
...
   place: string;
+  // Type of place shown in the chart if it's a chart that shows children places
   enclosedPlaceType: string;
```
Comment (kmoscoe) [inline static/nodejs_server/types.ts:67]: same as above

File: static/nodejs_server/types.ts [hunk 4]
```diff
@@ -60,18 +60,29 @@ export interface TileResult {
...
+  // Details about the variables to show in the chart including their dcids,
```
Comment (kmoscoe) [inline static/nodejs_server/types.ts:68]: same as above: make DCID uppercase throughout

File: static/nodejs_server/types.ts [hunk 5]
```diff
@@ -60,18 +60,29 @@ export interface TileResult {
...
+  // type id and the value is the spec for that event type.
   eventTypeSpec: Record<string, EventTypeSpec>;
 }
 
 // Debug info to return in /nodejs/query response
 export interface DebugInfo {
   timing: {
+    // The time it took to generate the chart configurations for a given query
     getNlResult: number;
+    // time it took to generate a results given the chart configurations
```
Comment (kmoscoe) [inline static/nodejs_server/types.ts:83]:
```suggestion
    // The time it took to generate results given the chart configurations
```

File: static/nodejs_server/types.ts [hunk 6]
```diff
@@ -60,18 +60,29 @@ export interface TileResult {
...
     getTileResults: number;
+    // total time it took to get from query to the results
```
Comment (kmoscoe) [inline static/nodejs_server/types.ts:85]:
```suggestion
    // The total time it took to get from query to the results
```

**Review by kmoscoe — APPROVED** [review]

**Comment by chejennifer** [comment]
thanks for the review!

**Review by beets — APPROVED** [review]
thanks for adding the documentation!

**Pull request merged**